### PR TITLE
Mus 116 stop button on play

### PR DESCRIPTION
**Pull request by souljaboy764**
I have made modifications to the code based of the feedback given

### CODE REVIEW

File: musicdroid/musicdroid.iml [hunk 1]
```diff
@@ -31,11 +31,13 @@
       <sourceFolder url="file://$MODULE_DIR$/build/generated/source/buildConfig/debug" isTestSource="false" generated="true" />
```
Comment (bteufl) [inline musicdroid/musicdroid.iml:31]: .iml files must not be committed

File: musicdroidSourceTest/musicdroidSourceTest.iml [hunk 2]
```diff
@@ -17,9 +17,9 @@
     <output-test url="file://$MODULE_DIR$/build/classes/test" />
```
Comment (bteufl) [inline musicdroidSourceTest/musicdroidSourceTest.iml:17]: please no iml files

**Comment by AndrejMueller01** [comment]
Don't change the following files:

.idea/encodings.xml
.idea/misc.xml
Musicdroid.iml
musicdroid/musicdroid.iml
musicdroidSourceTest/musicdroidSourceTest.iml

File: musicdroidSourceTest/musicdroidSourceTest.iml [hunk 3]
```diff
@@ -18,8 +18,8 @@
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src/main/resources" type="java-resource" />
-	  <sourceFolder url="file://$MODULE_DIR$/src/main/java" isTestSource="false" />
-	  <sourceFolder url="file://$MODULE_DIR$/src/test/resources" type="java-test-resource" />
+      <sourceFolder url="file://$MODULE_DIR$/src/main/java" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test/resources" type="java-test-resource" />
       <sourceFolder url="file://$MODULE_DIR$/src/test/java" isTestSource="true" />
       <excludeFolder url="file://$MODULE_DIR$/.gradle" />
       <excludeFolder url="file://$MODULE_DIR$/build" />
```
Comment (bteufl) [inline musicdroidSourceTest/musicdroidSourceTest.iml:25]: Please don't check in this file!!

Comment (souljaboy764) [inline musicdroidSourceTest/musicdroidSourceTest.iml:25]: I undid the changes that I had made by mistake. These are the original .iml files.

**Comment by souljaboy764** [comment]
Could you please tell me what changes I should make so that I make this something that can be merged with the main code?

**Comment by bteufl** [comment]
It is not possible to automatically merge this Request. Please merge the actual develop branch into your project and update your Pullrequest

**Comment by souljaboy764** [comment]
I have updated this with the development branch, is it possible to merge it now??

**Pull request closed without merging**